packages/documentation/src/stories/utilities/sizing/sizing.stories.ts

```diff
@@ -66,7 +78,10 @@ type Story = StoryObj;
 function renderSizing(args: Args) {
   const maximumHeight = args.maxHeight && args.maxHeight !== 'null' ? `mh-${args.maxHeight}` : '';
```

Comment:
This is named "none" in the options (same for all classes bellow) 
```suggestion
  const maximumHeight = args.maxHeight && args.maxHeight !== 'none' ? `mh-${args.maxHeight}` : '';
```